### PR TITLE
libnl: add feedstock

### PR DESCRIPTION
**Pull request by nehaljwani**

### CODE REVIEW

File: recipes/libnl/meta.yaml [hunk 1]
```diff
@@ -0,0 +1,49 @@
...
+    - make
+
+test:
+  commands:
+    - test -f $PREFIX/lib/libnl.so
+    - test -f $PREFIX/include/netlink/types.h
+
+about:
+  home: https://www.infradead.org/~tgr/libnl/
+  license: LGPL 2.1
```
Comment (chrisburr) [inline recipes/libnl/meta.yaml]: From [here](https://opensource.org/licenses/alphabetical):
```suggestion
  license: LGPL-2.1
```

File: recipes/libnl/meta.yaml [hunk 2]
```diff
@@ -0,0 +1,49 @@
...
+  license_file: COPYING
+  license_family: GPL
+  summary: "Netlink Library Suite"
+  description: |
+    The libnl suite is a collection of libraries providing APIs to netlink
+    protocol based Linux kernel interfaces. Netlink is a IPC mechanism primarly
+    between the kernel and user space processes. It was designed to be a more
+    flexible successor to ioctl to provide mainly networking related kernel
+    configuration and monitoring interfaces.
+  dev_url: github.com/tgraf/libnl-1.1-stable.git
```
Comment (chrisburr) [inline recipes/libnl/meta.yaml]: Do you plan to update this to version 3 after version 1 has been deployed?
```suggestion
  dev_url: https://github.com/thom311/libnl
```

Comment (nehaljwani) [reply recipes/libnl/meta.yaml]: Will change after updating it to 3.x the one provided you doesn't have tags for 1.x

Comment (chrisburr) [reply recipes/libnl/meta.yaml]: In that case the URL should be:
```suggestion
  dev_url: https://github.com/tgraf/libnl-1.1-stable
```

**Pull request merged**